### PR TITLE
Views

### PR DESCRIPTION
**Pull request by jaamanu**
In this PR
I queried the database to display the content to the frontend

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+                <%= image_tag user.photo %>
+            </div>
+            <div class='border flex'>
+                <%= link_to user.name, user %>
+                <div class='bottom_flex'>
+                    <%= "Number of posts: #{user.post_counter}" %>
+                </div>
+            </div>
+        </div>
+    <% end %>
```
Comment (MeqdamAlqudah) [inline app/views/users/index.html.erb:17]: - [ ] Please fix the width and the height of the images placeholder it should match the design it should have a border around the image.

- [ ] Please the name of the user should have the same font color as in the design 
design 
image
your code 
image

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+            <div class='bottom_flex'>
+                <%= "Number of posts: #{@user.post_counter} " %>
+            </div>
+        </div>
+    </div>
+    <div class='border'>
+        <p>Bio</p>
+        <%= @user.bio %>
+    </div>
+
```
Comment (MeqdamAlqudah) [inline app/views/users/show.html.erb:19]: - [  ] Please fix the width and the height of the images placeholder it should match the design it should have a border around the image.

- [ ] Please see all posts button should have a shadow
- [ ] Please the boxes should have margins as in the design
image
the design 
image
image

**Review by Reem-lab — COMMENTED** [review]
Hi @jaamanu, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌❌Invalid Code Review Request

You have submitted a project, that doesn't have the right link think this is for views but you submit it in add-form milestone kindly submit it in the right mile stone and also you submit a link with not updated README file so kindly update it with your Project information  and then you can send the PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,2 +1,43 @@
-<h1>users</h1>
-<p> Shows info of a single users</p>
\ No newline at end of file
+<div class='center'>
```
Comment (MeqdamAlqudah) [inline app/views/users/show.html.erb:1]: - [ ] Please fix the margins and padding in the post index page there is no space between the boxes, also the likes and comments counter should be aligned to the right and they should be in one line.
your code 
image
design 
image

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -1,2 +1,43 @@
...
+    <div class="flex">
```
Comment (MeqdamAlqudah) [inline app/views/users/show.html.erb:2]: - [ ] Please add a pagination button as in the design
image

File: app/views/posts/show.html.erb [hunk 5]
```diff
@@ -1,2 +1,30 @@
-<h1>Posts</h1>
-<p> Shows info of a single post</p>
\ No newline at end of file
+<div class='center2'>
+    <div class='border'>
+        <div class='flex jstSbt'>
+            Post by <%= @post.author.name %>
+            <div>
+                Comments : <%= @post.comment_counter %>,
+                Likes : <%= @post.likes_counter %>
```
Comment (MeqdamAlqudah) [inline app/views/posts/show.html.erb:7]: -  [ ]  Please add a margin to the top of posts to show page
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @jaamanu 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

image

 ## Highlights
:+1: no linter errors 
:+1: Readme file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**